scanner/src/test/kotlin/scanners/fossid/FossIdTest.kt

```diff
@@ -646,6 +646,18 @@ class FossIdTest : WordSpec({
             coVerify(exactly = 0) {
                 service.runScan(USER, API_KEY, scanCode, any())
             }
+
+            coVerify(exactly = 1) {
+                service.runScan(
+                    USER,
+                    API_KEY,
+                    scanCode,
+                    *arrayOf(
+                        "auto_identification_detect_declaration" to "0",
+                        "auto_identification_detect_copyright" to "0"
+                    )
```

Comment:
Sorry, I'm not good at reading MockK code. While I've verified that the test passes, I do not understand why. Doesn't this array also match the `any()` from line 647? Because if so, how can line 647 not be called at all, but this code be called once?

Reply:
> doesn't this array also match the any() from line 647? 

I thought that too initially, but I found it doesn't.

Reply:
Apparently one has to use [`*anyVararg()`](https://mockk.io/#varargs) instead.